tests/testthat/test-are-all-close.R

```diff
@@ -0,0 +1,51 @@
+test_that("are_all_close_correct_true", {
+
+  # 1) correctly returns TRUE
+
+  abs_tol = 1e-6
+  rel_tol = 1e-6
+
+  v = rnorm(5)
```

Comment:
-2

Make sure any randomness in tests is seeded; unit tests should be deterministic. In this case, the randomness doesn't really help, just use a specific example like c(1,1). Unit tests should use simple test cases that are as easy to read and verify as possible.